dev/online-editors/codesandbox/__transforms__/app.mjs

```diff
@@ -0,0 +1,68 @@
+import * as path from 'node:path';
+import fse from 'fs-extra';
+import { js } from 'ember-apply';
```

Comment:
I used my own tool here because the ceramony of dealing with loading and parsing a file with intent to transform is a bit verbose -- docs here: 

https://ember-apply.pages.dev/functions/js.transform


I use `ember-apply` mostly for monorepo stuff, but I've also been using it to automatically apply changes to deprecation-workflow files: https://ember-apply.pages.dev/functions/ember.deprecations.withAST

and then combined with other tooling, I'd assign work to other teams to fix their deprecation violations.

maybe others will find that useful? idk.
Here, in ember-cli, it's just a devDependency -- this comment is more FYI, I guess, if folks manage lots of projects